exec/csnex/Exec.cs

```diff
@@ -278,7 +322,8 @@ void DIVN()
             Number b = stack.Pop().Number;
             Number a = stack.Pop().Number;
             if (b.IsZero()) {
-                throw new NeonDivideByZeroException("DivideByZeroException", "");
```

Comment:
If the class `NeonDivideByZeroException` is no longer used, is it still defined somewhere? Shouldn't that declaration be removed now?

Reply:
Removed unused exceptions.